target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -510,7 +510,9 @@ ramips_setup_macs()
 	alfa-network,w502u|\
 	arcwireless,freestation5|\
 	cudy,wr1000|\
-	netgear,wnce2001)
+	netgear,wnce2001|\
+	wevo,11acnas|\
+	wevo,w2914ns-v2)
 		wan_mac=$(mtd_get_mac_binary factory 0x2e)
```

Comment:
The looks strange to me, as &ethernet pulls the address from <&factory 0xe000>.

Can you provide a hexdump of the whole factory partition (for both devices if possible)?

Reply:
I'm not sure I can post the whole partition here.
The MAC addresses of 11acnas device I have are xx:xx:xx:99:xx:yy,
and I searched all "99" occurrences in the factory partition. Then I got:
0x4: xx:xx:xx:99:xx:80 (WiFi 2.4GHz)
0x2e: xx:xx:xx:99:xx:89 (WAN and LABEL)
0x8004: xx:xx:xx:99:xx:84 (WiFi 5GHz)
0xe000: xx:xx:xx:99:xx:88 (LAN)

I don't have w2914ns-v2 device, so it needs a test from someone else. But AFAIK the two devices are almost same, so I think it will work for w2914ns-v2 too.

Reply:
Okay, can you check 0x28 and 0xe006 on your device, too?

Reply:
It's blank.
0x28: fd:9b:ff:ff:ff:ff
0xe006: ff:ff:ff:ff:ff:ff

Reply:
Okay, they really have a strange setup there, but what you did looks correct to me. Thank you for the additional research.

Would still be interested in the same information about w2914ns-v2 if someone can get it.

Reply:
OK, I'll grab my factory of W2914NS-V2 today and will post the result of the analysis. today.

Reply:
It is same as above result.

Reply:
Great. That's the answer I've been waiting for. Thanks! :D

Reply:
Okay, so this is settled. Since you have tested this, I would add this information to the commit message, like:
"
Both devices have an untypical MAC address configuration (both tested!):
0x4: xx:xx:xx:xx:xx:80 (WiFi 2.4GHz)
0x28: fd:9b:ff:ff:ff:ff (unused)
0x2e: xx:xx:xx:xx:xx:89 (WAN and label)
0x8004: xx:xx:xx:xx:xx:84 (WiFi 5GHz)
0xe000: xx:xx:xx:xx:xx:88 (LAN)
0xe006: ff:ff:ff:ff:ff:ff
"

Reply:
Okay, added.